web/client/configs/simple.json

```diff
@@ -76,6 +76,13 @@
       }
     ],
     "desktop": [
+      {
+        "name": "DynamicLegend",
+        "cfg": {
+          "isFloating": false,
+          "flatLegend": false
+        }
+      },
```

Comment:
This plugin should be available only in contexts so it should be removed from simple.json